config/crd/bases/operator.knative.dev_knativeservings.yaml

```diff
@@ -408,6 +408,145 @@ spec:
                             type: string
                         type: object
                       type: array
+                    topologySpreadConstraints:
```

Comment:
Please add the same change to `spec.deployments` as well.
At this moment, spec.deployments and spec.workloads co-exist.

Reply:
Implemented.